### ISSUE TITLE
Logger does not correctly determine the package key

### ISSUE THREAD

**Issue by neos-bot** [neos/flow-development-collection#558]
Jira issue originally created by user @astehlik:

The automatic detection of the package key in the Logger class does not work correctly.

There is already a FIXME annotation in the log() method:

// FIXME: This is not really the package key:

The detection of the actual package key needs to be improved.

 Jira-URL: https://jira.neos.io/browse/FLOW-226

**Comment by RafaelKa** [comment]
What ablout using composer informations to get correct package key?

[Composer\Autoload\ClassLoader->getPrefixes() 
and 
Composer\Autoload\ClassLoader->getPrefixesPsr4()](https://github.com/composer/composer/blob/2853e82400a113620e05504c52729c61b5a53f77/src/Composer/Autoload/ClassLoader.php#L59-L71)
Could provide us required infos to resolve correct package key. -> [here is what we can get from composer](https://gist.github.com/RafaelKa/cc1e3f13580b282b89eaf19cd9a208c4)

Additional Infos:
[link to // FIXME: This is not really the package key:](https://github.com/neos/flow-development-collection/blob/b7788469e27f33ff7f912fc211ec1e358478d958/TYPO3.Flow/Classes/TYPO3/Flow/Log/Logger.php#L94)

**Comment by kitsunet** [comment]
I rather would refrain from providing the notion of a package key at all. The class name is provided and with that it should be easy enough to figure out the package.
So in that regard I would like to rephrase this to removing the package from the log messages.

**Comment by daniellienert** [comment]
Fixed with #1399 and #1408
